plugins/shared-react/src/components/common/Status.tsx

```diff
@@ -59,37 +62,43 @@ const useStatusStyles = makeStyles(theme => ({
   },
 }));
 
-const StatusIcon = ({ statusKey }: { statusKey: StatusClassKey }) => {
+const StatusIcon = ({
+  statusKey,
+  classNames,
+}: {
+  statusKey: StatusClassKey;
+  classNames: string;
```

Comment:
Sorry I missed this on the previous review.

1. isn't `className` more common?
2. this should be optional, otherwise it could break existing code (at compile time)

```suggestion
  className?: string;
```